src/user-provisioning-service.js

```diff
@@ -0,0 +1,35 @@
+const dotenv = require('dotenv');
+const authApiProxy = require('./proxies/authenticateapi');
+const provision = require('./provision');
+
+//Used for the express HTTP listener
+const express = require('express');
+const bodyParser = require('body-parser');
+const {response} = require('express');
```

Comment:
```suggestion
const {response} = express;
```

Reply:
on second thought this isn't used anywhere.

Reply:
Was leftover from some earlier code I was playing around with then removed.